metrics/frugalscore/README.md

```diff
@@ -0,0 +1,103 @@
+# Metric Card for FrugalScore
+
+
+## Metric Description
+FrugalScore is a reference-based metric for NLG models evaluation. It is based on a distillation approach that allows to learn a fixed, low cost version of any expensive NLG metric, while retaining most of its original performance.
+
+## How to use 
+
+When loading FrugalScore, you can indicate the model you wish to use to compute the score. The default model is `tiny_bert-base_bert-score`, and a full list of models can be found in the [Limitations and bias](#Limitations-and-bias) section.
+
+```python
+>>> from datasets import load_metric
+>>> frugalscore = load_metric("frugalscore", checkpoint = "moussaKam/frugalscore_medium_bert-base_mover-score")
```

Comment:
Bug: if you pass `checkpoint` arg, this is ignored. I have tried your code and the metric's model and tokenizer were using the default `'moussaKam/frugalscore_tiny_bert-base_bert-score'`:
```python
In [2]: frugalscore = load_metric("frugalscore", checkpoint="moussaKam/frugalscore_medium_bert-base_mover-score")
In [3]: frugalscore.tokenizer
Out[3]: PreTrainedTokenizerFast(name_or_path='moussaKam/frugalscore_tiny_bert-base_bert-score', vocab_size=30522, model_max_len=1000000000000000019884624838656, is_fast=True, padding_side='right', special_tokens={'unk_token': '[UNK]', 'sep_token': '[SEP]', 'pad_token': '[PAD]', 'cls_token': '[CLS]', 'mask_token': '[MASK]'})
```

```suggestion
>>> frugalscore = load_metric("frugalscore", "moussaKam/frugalscore_medium_bert-base_mover-score")
```

```python
In [6]: frugalscore = load_metric("frugalscore", "moussaKam/frugalscore_medium_bert-base_mover-score")

In [7]: frugalscore.tokenizer
Out[7]: PreTrainedTokenizerFast(name_or_path='moussaKam/frugalscore_medium_bert-base_mover-score', vocab_size=30522, model_max_len=1000000000000000019884624838656, is_fast=True, padding_side='right', special_tokens={'unk_token': '[UNK]', 'sep_token': '[SEP]', 'pad_token': '[PAD]', 'cls_token': '[CLS]', 'mask_token': '[MASK]'})

Reply:
haha so, do we fix the bug, or leave as is?